aioautomower/session.py

```diff
@@ -189,6 +187,11 @@ async def _ws_task(self):
         printed_err_msg = False
         async with aiohttp.ClientSession() as session:
             while True:
+                if self.token["scope"] != "iam:read amc:api":
```

Comment:
"amc:api" not in self.token["scope"]

Reply:
Move this check to the connect method since we know that it's pointless to start the _ws_task if `amc:api` is not available